pkg/webhook/admission/inferenceservice/inferenceservice_validator.go

```diff
@@ -43,6 +48,24 @@ func (validator *Validator) Handle(ctx context.Context, req admissiontypes.Reque
 		return admission.ErrorResponse(http.StatusInternalServerError, err)
 	}
 
+	ns := &v1.Namespace{}
+	if err := validator.Client.Get(context.TODO(), ktypes.NamespacedName{Name: req.AdmissionRequest.Namespace}, ns); err != nil {
+		return admission.ErrorResponse(http.StatusInternalServerError, err)
+	}
+	validNS := true
+	if ns.Labels == nil {
+		validNS = false
+	} else {
+		if v, ok := ns.Labels[constants.InferenceServicePodLabelKey]; !ok || v != constants.EnableKFServingMutatingWebhook {
+			validNS = false
```

Comment:
Good to have this which should solve issue #568 for now since we cannot enforce object selector before kubernetes.  I think this will have wide impact for existing example and test, we need label the namespace firstly to make test pass. And also should we document and add the prerequest to label the namespace?